main-process/packages/blockchain-wallet-v4-frontend/package.json

```diff
@@ -88,6 +88,7 @@
     "@ledgerhq/hw-app-btc": "4.30.0",
     "@ledgerhq/hw-app-str": "4.26.0-beta.ebeb3540",
     "@ledgerhq/hw-transport-u2f": "4.31.0",
+    "@nodeguy/channel": "0.6.4",
```

Comment:
In the interest of reducing external dependencies, would it be better for @blockchain to fork this or create an internal service.

Reply:
agreed, I've once encountered a problem where personal namespaces drifted off from company needs. Double the support prs but ownership is important

Reply:
@guy-blockchain We currently have 387 production dependencies.  I think it will be an interesting conversation in the future to explore how to minimize dependencies but for now it's not a big deal to add 2 more.

@tony-blockchain I don't understand your point.  We don't own most of the 387 dependencies (e.g., React).  How is this one any different?